code/modules/asset_cache/validate_assets.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	</head>
+	<body>
+		<script>
+			//this is used over window.location because window.location has a character limit in IE.
+			function sendbyond(text) {
+				var xhr = new XMLHttpRequest();
+				xhr.open('GET', '?'+text, true);
+				xhr.send(null);

```

Comment:
Daily reminder that this works since 511. as does `$.send()` or what ever it is in jquery.